lib/xandra/protocol.ex

```diff
@@ -306,6 +306,12 @@ defmodule Xandra.Protocol do
     <<date::32>>
   end
 
+  defp encode_value(:decimal, float) when is_float(float) do
+    encode_value(:decimal, Decimal.new(float))
+  end
```

Comment:
@ericmj if you don't think it's convenient I will remove it; just sometimes you have a float that you want to put into a decimal column.  It could allow for evil mistakes so on balance you are correct.